### ISSUE TITLE
Wav2Vec: loss curve

### ISSUE THREAD

**Issue by Oktai15** [pytorch/fairseq#1661]
Could you provide loss curve that you get during training wav2vec?

**Comment by sgxthsat2019** [comment]
Hi,
Did you handle this? I want to plot the loss curve, too. Thank you!

**Comment by sgxthsat2019** [comment]
@Oktai15

**Comment by Oktai15** [comment]
@sgxthsat2019 unfortunately, no, I didn’t resolve problem

**Comment by MarvinLvn** [comment]
Hi both !

I'm working on VQ-wav2vec, but adding the option `--tensorboard-logdir /path/to/tensorboard_logdir` should solve your problem.
Then, you'll just have to call `tensorboard --logdir=/path/to/tensorboard_logdir` to display the tensorboard interface, showing you the validation loss (amongst other things)

I haven't found the way to store this information under a `.json` file format or something similar though ...
